### PR TITLE
Adds a few styles of yukata.

### PR DESCRIPTION
**Pull request by DisturbHerb**
[FEATURE][CLOTHING]

## About the PR 
This pull requests adds yukata, a type of light Japanese garment worn in the summer, to the game's code. These can be purchased from Cargo alongside pairs of sandals and fireworks as part of the Obon Festival Crate, costing 3000 credits.

There are 10 different styles, divided into 5 plain and 5 floral yukata. In-hands exist for them all, but monkey versions don't currently exist yet.
dondonpanpan

Yes, this is all a reference to Rhythm Heaven. Sorry.
mistake

### To do
- [x] Clothing booth entries.
- [ ] Maybe add some hairclips to go along with them.

## Why's this needed? 
Thought they were cute.

## Changelog 

```changelog
(u)DisturbHerb
(*)Added the Obon Festival Crate to QM, containing traditional summer robes and fireworks for the observance of the Space Japanese Obon festival. Costs 3000 credits.
```

### CODE REVIEW

**Review by flrsh — CHANGES_REQUESTED** [review]
I don't think these should be added to the clothing booth. First, this adds a lot of unnecessary bloat, second, I don't think we should be adding yukatas as costumes for people to wear. The vast majority of our players' exposure to and understanding of Japanese culture is limited to just anime / weeb stuff and there's a very long history of people in general being weird and fetishizy. This is also why I'm against things like "Syndicate ninjas". I understand that you don't have any bad intentions but I'm still uncomfortable with the idea.

**Comment by DisturbHerb** [comment]
By no means am I particularly firm on keeping these in the booth - being myself keenly aware of the sorry state that it is in - so long as some method remains for players to acquire them. I think it'd be plenty easy to switch to something else.

With regard to your other concerns, I am also cognizant of the fact that an open pull request poses zero guarantees of the content being merged and that I absolutely will be subject to - quite often firm - feedback and scrutiny. While I won't be disputing your valid discomfort at the inclusion of this content in the codebase, the point being raised has been unusually disheartening to hear, knowing that anything I had a hand in creating could inspire such feelings; to where I have found it difficult to draft an appropriate response. 

Should this represent a consensus among the development team, this does not bode well for any of the other pieces of content I've created. I'll have to make my peace with that if it comes to it, but it is something that I would find very unfortunate; especially given that these issues aren't the result of any faults from either of us.

**Comment by flrsh** [comment]
My opinion is not representative of the dev team, as evidenced by Leah's react on my post. I think I may be in the minority here after discussing this with some other admins.

Personally I'd be more fine if there were also context that you'd have to acknowledge ingame. For instance, a custom prefab or with a trader who tells you about yukata. I'm not a fan of the clothing booth in general, but even something like a description section that gives you more info about their history and who wears them where/when/how. Though that'd require TGUI work and I'd still have my issue about bloat. 

I still am very ambivalent about this, but given that most people don't seem to share my opinion, I'd just appreciate a compromise.

**Comment by DisturbHerb** [comment]
> I'm not a fan of the clothing booth in general, but even something like a description section that gives you more info about their history and who wears them where/when/how. Though that'd require TGUI work and I'd still have my issue about bloat. And it'd still feel weirder than the other options (happy to elaborate if needed).

I'd like to hear your further thoughts on this. Having the clothing booth provide a description of each item wouldn't be a difficult thing to implement, I think, it'd just require me to individually edit each database item we've presently got. Other solutions still remain available.

**Comment by flrsh** [comment]
Sure yeah. I think why the clothing booth option feels more uncomfortable than the other potential options is that we present it as much more of a disembodied commodity. 

Even if you had a blurb saying "yeah, these are light cotton robes that were traditionally worn as bathrobes, nowadays Japanese people typically wear them during the summer, especially at festivals, ..." or whatever description you wanted to put, there's not really a connection between the clothing item and its actual context, and it feels like we're adding it and people would be wearing it solely for its aesthetic without regard for anything else. 

Of course this _already_ happens with everything else in the clothing booth, but also I'm pretty sure those items aren't particular to a certain culture in the way that a yukata is. Though if there are (not totally sure cuz there are so many things and I haven't kept track), I also think that should be rectified. 

And I know that, obviously, in the real world, yukata _are_ commodified, and you can just get one on Amazon or whatever without caring about anything besides "oh I saw this in an anime and I want to wear one now", and that some people (including Japanese people) would even find a _good_ thing because [insert many possible reasons such as normalizing wearing this kind of clothing, spreading Japanese culture, furthering Japan's soft power, etc.], but - it's very complicated and I'd rather us not re-enact this when we don't have to. (Will also elaborate on this if you'd like, but this comment is also super long already so.)

Re: alternatives, I feel like something like coming across a prefab that does set up a certain scene and compel you to internalize some amount of context before you retrieve and put on this clothing item, or meeting a trader who tells you about how they got this from a friend, or if they made it themselves, and how there's a fly-in fireworks festival next month or whatever, before they sell it to yourself, would feel significantly less disembodied. Or a QM crate with all ten styles with accompanying items such as fireworks and sandals that make it somewhat more like a "whole" rather than just a displaced item of clothing. Which would also keep the opportunity for many people to easily wear them for a special occasion.

**Comment by DisturbHerb** [comment]
Removed the clothing booth entries, yukata are now part of an Obon Festival Crate (along with other relevant items) available from QM for 3000 credits.

File: code/modules/vending/clothingbooth_items.dm [hunk 1]
```diff
@@ -619,13 +619,9 @@ ABSTRACT_TYPE(/datum/clothingbooth_item/outerwear)
...
 
 /datum/clothingbooth_item/outerwear/jean_jacket
-	name = "Jean Jacket"
+	name = "Jean Jackett"
 	path = /obj/item/clothing/suit/jean_jacket
 
-/datum/clothingbooth_item/outerwear/jacketsjacket
-	name = "Baseball Jacket"
-	path = /obj/item/clothing/suit/jacketsjacket
-
```
Comment (DimWhat) [inline code/modules/vending/clothingbooth_items.dm:628]: Ditto.

File: code/modules/vending/clothingbooth_items.dm [hunk 2]
```diff
@@ -619,13 +619,9 @@ ABSTRACT_TYPE(/datum/clothingbooth_item/outerwear)
 	path = /obj/item/clothing/suit/merchant
 
 /datum/clothingbooth_item/outerwear/jean_jacket
-	name = "Jean Jacket"
+	name = "Jean Jackett"
```
Comment (DimWhat) [inline code/modules/vending/clothingbooth_items.dm:622]: This doesn't seem like an intentional change.

**Comment by Sovexe** [comment]
changelog?

**Comment by DisturbHerb** [comment]
> changelog?

how could i possibly forget :waa:

**Pull request merged**